phlib/include/phnative.h

```diff
@@ -3821,7 +3821,8 @@ NTSTATUS
 NTAPI
 PhGetProcessSystemDllInitBlock(
     _In_ HANDLE ProcessHandle,
-    _Out_ PPS_SYSTEM_DLL_INIT_BLOCK SystemDllInitBlock
+    _In_ ULONG RequiredSize,
+    _Out_writes_bytes_(RequiredSize) PPS_SYSTEM_DLL_INIT_BLOCK SystemDllInitBlock
```

Comment:
Should this include `_Post_readable_size_(RequiredSize)`?

Reply:
Yes, good idea. Although, it makes me wonder why one doesn't automatically follow form the other.